app/models/user.rb

```diff
@@ -4,7 +4,7 @@ class User < ActiveRecord::Base
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable,
          :omniauthable, omniauth_providers: [:twitter],
-         :authentication_keys => [:login]
+         authentication_keys: [:login]
```

Comment:
__[rubocop]__
* Style/AlignHash: Align the elements of a hash literal if they span more than one line.